service/history/workflow/mutable_state_impl.go

```diff
@@ -2317,8 +2334,13 @@ func (ms *MutableStateImpl) ApplyWorkflowExecutionStartedEvent(
 	}
 
 	ms.approximateSize -= ms.executionInfo.Size()
+	ms.approximateSize -= ms.executionState.Size()
```

Comment:
I think this is a bug in the mutable approximate size. In L2339, it sets the `CreateRequestId`, but it's not updating the size.
So, I added this line and L2474 to correct the approximate size.

cc: @yycptt

Reply:
Thanks!